### PR TITLE
dash: removing usage of local storage temporarily

### PR DESCRIPTION
**Pull request by jdslaugh**
### Description
Issue with deprecated projects is causing alarms with attempting to load from the stored state. To get around this we are temporarily bypassing the loading of the stored state and instead will just start fresh.

### Testing Performed
Manual

### CODE REVIEW

**Review by dschaller — APPROVED** [review]

File: frontend/workflows/projectSelector/src/project-selector.tsx [hunk 1]
```diff
@@ -92,12 +92,8 @@ const hydrateProjects = (state: State, dispatch: React.Dispatch<Action>) => {
       users: string[];
       projects: string[];
     };
-    _.forEach(Object.keys(state[Group.PROJECTS]), p => {
-      // if the project is custom
-      if (state[Group.PROJECTS][p].custom) {
-        requestParams.projects.push(p);
-      }
-    });
```
Comment (dschaller) [inline frontend/workflows/projectSelector/src/project-selector.tsx:100]: minor nit: maybe do this in another PR since it's unrelated?

**Review by danielhochman — APPROVED** [review]

File: frontend/workflows/projectSelector/src/project-selector.tsx (hunk 1 above)
Comment (jdslaugh) [reply frontend/workflows/projectSelector/src/project-selector.tsx]: Adjusted
